content/en/database_monitoring/guide/aurora_autodiscovery.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Automatically configure your aurora clusters for monitoring
+kind: guide
+---
+
+{{< beta-callout url="#" btn_hidden="true" >}}
+Aurora cluster auto discovery is in **beta**. If you have any feedback about this feature, please [contact support](https://docs.datadoghq.com/help/).
+{{< /beta-callout >}}
+
+This guide assumes that you have configured [Database Monitoring][1] for your Aurora hosts.
+
+## Before you begin
+
+Supported databases
+: Postgres, MySQL
+
+Supported Agent versions
+: 7.53.0+ (Not currently released)
+
+## Agent Installation
+
+// TODO: Add the installation steps for the beta agent
+
+## Overview
+
+The Datadog Agent supports configuring automatic discovery and monitoring of your aurora clusters. This feature works in lieu of manually configuring the Agent by listing individual database host endpoints.
+
+This is helpful for environments where the number of endpoints can be dynamic, such as with clusters that use [Aurora Auto Scaling][6].
+
+This features leverages the Agent [Autodiscovery feature][4] and lets you define configuration templates for Postgres or MySQL checks, and specify which clusters each check should apply to.
+
+## How it works
+
+This utilizes an Agent service listener, responsible for discovering all database host endpoints in an aurora cluster and forwarding discovered endpoints to the existing Agent check scheduling pipeline. You can configure the listener in the `datadog.yaml` file:
+
+```yaml
+database_monitoring:
+  autodiscovery:
+    aurora:
+      enabled: true
+```
+
+By default, the listener will discover all aurora clusters in the account and region where the Agent is running that have the `datadoghq.com/scrape:true` tag applied. You can also configure the listener to discover clusters with specific tags.
+
+Specify custom tags for the aurora cluster discovery in the `datadog.yaml` file:
```

Comment:
should we link an aws doc on how to add/modify tags for aurora?

Reply:
yes, i'll do that